CONTRIBUTION.md

```diff
@@ -52,6 +62,8 @@ In general apply common sense to the Code you write. "Effective Java" is also al
 * Use Guava's `Preconditions` class for Argument validation
 * Validate `@NonNull` values with `Objects.requireNonNull` if necessary
 * use getters and setters, even in the declaring class
+* avoid getters and setters for protected methods
+* use protected methods where external classes do not require access or in instances where you have a private var without getters or setters.
```

Comment:
I'm unsure what you're referring to here actually - In General access should be as limited as possible...? 

package-private Methods or classes are also feasible things...

Reply:
Effectively use the right level of access

Reply:
```suggestion
* In general try to use the smallest visibility-scope possible, expcept if it's likely that others will want access to a given method.
```